app/src/main/java/org/openobservatory/ooniprobe/client/OONIAPIClient.java

```diff
@@ -0,0 +1,12 @@
+package org.openobservatory.ooniprobe.client;
+
+import org.openobservatory.ooniprobe.model.api.ApiMeasurement;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface OONIAPIClient {
+	@GET("api/v1/measurements")
```

Comment:
I am wondering whether this should be `/api/v1/...`. A relative path is a bit surprising to me.

Reply:
It's how retrofit works, link to doc https://square.github.io/retrofit/

Reply:
So, this was a reliability question. If the base URL has no final `/`, what happens? (I am saying we need to understand whether this is something to keep in mind.)

Reply:
I couldn't find anything more on this page, I just used the basic guide to retrofit and you are not supposed to add the final /

Reply:
Yeah. I am asking whether you could try and see whether it works adding the initial `/` to the URL path and/or it works removing the final `/` to the base URL. Just so we know what could possibly break this mechanism. I'm fine with leaving the code as compliant as possible with retrofit's docs, which is what we have now.

Reply:
I researched this topic. Retrofit will accept `/` everywhere. When you do so, the path is considered being absolute and hence the base URL path is discarded, e.g.:

> baseUrl(http://www.example.com/foo) + GET(/bar/baz) => http://www.example.com/bar/baz

Conversely, when the path is relative:

> baseUrl(http://www.example.com/foo) + GET(bar/baz) => http://www.example.com/foo/bar/baz

Source: https://github.com/square/retrofit/issues/2075